.sbtopts

```diff
@@ -0,0 +1,2 @@
+-J-Xmx2G
```

Comment:
@thesamet 

Was getting `OutOfMemoryError` without `-Xmx` on my computer and even on Action runner. Added this for the project, if you don't mind.